packages/@aws-cdk/toolkit/lib/toolkit.ts

```diff
@@ -217,9 +217,9 @@ export class Toolkit {
       if (Object.keys(stack.template.Resources || {}).length === 0) {
         // The generated stack has no resources
         if (!(await deployments.stackExists({ stack }))) {
-          warning('%s: stack has no resources, skipping deployment.', chalk.bold(stack.displayName));
+          await ioHost.notify(warning(`${chalk.bold(stack.displayName)}: stack has no resources, skipping deployment.`));
         } else {
-          warning('%s: stack has no resources, deleting existing stack.', chalk.bold(stack.displayName));
+          await ioHost.notify(warning(`${chalk.bold(stack.displayName)}: stack has no resources, deleting existing stack.`));
```

Comment:
i prefer this to `%s` and it wasn't immediately obvious to me how to maintain that, hope this is still ok

Reply:
I removed support for `%s`. This is great.